internals/scripts/clean.js

```diff
@@ -27,27 +27,35 @@ rm('-rf', 'app/components/*');
 // Cleanup containers folder
 rm('-rf', 'app/containers/*');
 mkdir('app/containers/App');
-mkdir('app/components/NotFoundPage');
-mkdir('app/components/HomePage');
+mkdir('app/containers/NotFoundPage');
+mkdir('app/containers/HomePage');
```

Comment:
These should actually be in the components folder, as they don't have any associated state with them!